main.js

```diff
@@ -46,27 +47,48 @@ async function _handleResponse(target, thisArg, args) {
   return response;
 }
 
-async function _handleError(e) {
-  // handle network errors that do not have a response
-  if(!e.response) {
-    if(e.message === 'Failed to fetch') {
-      e.message = `${e.message}. Possible CORS error.`;
+/**
+ * @param {Error} error - Error thrown during http operation.
+ * @param {string} url - Target URL of the request.
```

Comment:
Needs `options` and `optons.foo` notation to be valid jsdoc.

Reply:
Fixed.